samples/BenchmarkDotNet.Samples/IntroPowerPlan.cs

```diff
@@ -0,0 +1,12 @@
+﻿using BenchmarkDotNet.Attributes;
+
+namespace BenchmarkDotNet.Samples
+{
+    [HighPerformancePowerPlan(false)]
+    public class IntroPowerPlan
```

Comment:
In the IntroSample, we need two jobs (enabled and disabled high performance power plan) to show the difference between them. An example of the summary table should be included in `IntroPowerPlan.md`.